linux_sql/scripts/psql_docker.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+#This is how to use the script:
+#./scripts/psql_docker.sh start|stop [db_password]
+
+command=$1
+dbpassword=$2
+
+case $command in
+	start)
+		#first check if command is in correct format (with password)
+		if [ "$#" -ne 2 ]
+		then
+			echo "Incorrect usage, please follow: psql_docker.sh start|stop [db_password]"
+			exit 1
+		fi
+
+		#then  we start docker if the docker server is not running
+		sudo systemctl status docker || sudo systemctl start docker
+
+		#next we check to see if our container is already running, if it is then simply exit
+		if [ `sudo docker ps -f name=jrvs-psql | wc -l` -eq 2 ]
+		then
+			echo "jrvs-psql container is already running"
+			exit 0
+		fi
+
+		#then get psql docker image
+		sudo docker pull postgres
+
+		#now we check if our volume was already created or not, if not then create it
+		if [ -z `sudo docker volume ls | egrep "pgdata" | awk '{print $2}'` ]
```

Comment:
if-check is not necessary since `docker volume create pgdata` does not recreate an exiting volume

Reply:
Just FYI, I will approve your PR

Reply:
thanks, will update